src/components/@shared/atoms/Modal/index.text.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import Modal from '@shared/atoms/Modal'
+import ReactModal from 'react-modal'
+
+describe('Modal', () => {
```

Comment:
sure, why not, but:

1. file is named wrong so test is not run
2. we already have a story, if that covers the component already, then goal was to use `ReactModal.setAppElement` on the story, not in an extra test. But if only this test can give full coverage then keep it